pkg/agent/controller/noderoute/node_route_controller.go

```diff
@@ -615,3 +621,30 @@ func GetNodeAddr(node *corev1.Node) (net.IP, error) {
 	}
 	return ipAddr, nil
 }
+
+func (c *Controller) IPInPodSubnets(ip net.IP) bool {
+	var ipCIDR *net.IPNet
+	var curNodeCIDRStr string
+	if ip.To4() != nil {
+		v4Mask := net.CIDRMask(PodIPv4CIDRMaskSize, IPv4BitLen)
+		ipCIDR = &net.IPNet{
+			IP:   ip.Mask(v4Mask),
+			Mask: v4Mask,
+		}
+		if c.nodeConfig.PodIPv4CIDR != nil {
+			curNodeCIDRStr = c.nodeConfig.PodIPv4CIDR.String()
+		}
+	} else {
+		v6Mask := net.CIDRMask(PodIPv6CIDRMaskSize, IPv6BitLen)
+		ipCIDR = &net.IPNet{
+			IP:   ip.Mask(v6Mask),
+			Mask: v6Mask,
+		}
+		if c.nodeConfig.PodIPv6CIDR != nil {
+			curNodeCIDRStr = c.nodeConfig.PodIPv6CIDR.String()
+		}
+	}
+	ipCIDRStr := ipCIDR.String()
+	nodesHaveSamePodCIDR, _ := c.installedNodes.ByIndex(nodeRouteInfoPodCIDRIndexName, ipCIDRStr)
```

Comment:
`nodeInCluster` is better name. I feel the context for `nodesHaveSamePodCIDR` is different.

Reply:
Thanks, addressed.